projects/1.16/assets/biomancy/biomancy/lang/zh_cn.json

```
@@ -0,0 +1,265 @@
+  "reagent.biomancy.absorption_boost": "伤害吸收剂",
+  "reagent.biomancy.insomnia_cure": "失眠治疗剂",
+  "reagent.biomancy.cleansing_serum": "净化血清",
+  "reagent.biomancy.decay_agent": "坏死血清",
+  "reagent.biomancy.mutagen_serum": "诱变血清",
+  "reagent.biomancy.blood_sample": "血液样本",
+
+  "attribute.generic.attack_distance": "攻击距离",
+
+  "tooltip.biomancy.unveiling_oculi": "赋予你细致的视线，揭示未曾所见之物。你不会再被黑暗或魔法致盲。",
+  "tooltip.biomancy.fleshborn_war_axe": "一把暴脾气的斧头，当持有者的生命值低于敌人最大生命值的 75% 时，有 60% 的几率造成敌人最大生命值的 30% 的伤害。",
+  "tooltip.biomancy.fleshborn_khopesh": "一把由斧头进化而来的镰刀剑。",
+  "tooltip.biomancy.blood_curse_axe": "持有者生命值越低，攻击速度越快的斧头。",
+  "tooltip.biomancy.leech_claw": "攻击会回复造成伤害 50% 的生命。\n还会在重击时获得 15 秒的额外生命，最多可叠加 5 层。",
+  "tooltip.biomancy.long_range_claw": "有着更长攻击距离的血肉之爪，使用重击可以激活爪子，再加长攻击距离。",
+  "tooltip.biomancy.fleshborn_guan_dao": "有着极好的攻击距离和适应性。对同一个目标的反复重击会提升它对该目标的伤害。",
+  "tooltip.biomancy.single_item_bag": "可以储存大量单一物品。\n吞噬模式下，它会尝试吃掉背包中相同的所有物品\n补充模式下，它会尝试补充你主手或副手上的方块。",
+  "tooltip.biomancy.entity_storage": "可以储存一个活着的实体在其胃袋中。",
+  "tooltip.biomancy.fleshborn_pickaxe": "可以同时挖掘3x3的方块（按住潜行禁用）只会挖掘相同的方块。",
+  "tooltip.biomancy.fleshborn_axe": "可以同时挖掘3x3的方块（按住潜行禁用）只会挖掘相同的方块。",
+  "tooltip.biomancy.fleshborn_shovel": "可以同时挖掘3x3的方块（按住潜行禁用）只会挖掘相同的方块。",
+  "tooltip.biomancy.oculus_key": "将所有者的血清与该钥匙合成之后，即可用来锁定方块。",
+  "tooltip.biomancy.injection_device": "允许你将试剂注入生物或块中（有些试剂不能被注入任何东西中）\n潜行单击可以装填/卸载副手玻璃瓶或试管中的液体。\n未装载液体时，可以从生物中提取血样。",
+  "tooltip.biomancy.tooth_gun": "射出小牙齿\n使用营养物质作为弹药。",
+  "tooltip.biomancy.withershot": "射出小而致命的凋灵骷髅头。当击杀目标后回复持有者的生命。\n使用凋灵骷髅头作为弹药。",
+  "tooltip.biomancy.boomling": "一种跟踪热源的药水装载生物。\n与喷溅型药水合成可以设置其药水效果。\n丢出后可以潜行右击收回。",
+  "tooltip.biomancy.reagent": "注射后会复原生物或方块的奇异物质。",
+  "tooltip.biomancy.growth_serum": "促进作物生长，包括甘蔗、仙人掌和地狱疣。",
+  "tooltip.biomancy.rejuvenation_serum": "还原植物或动物。",
+  "tooltip.biomancy.breeding_stimulant": "让动物立即准备好下次交配。",
+  "tooltip.biomancy.absorption_boost": "提供可堆叠的吸收生命值。",
+  "tooltip.biomancy.insomnia_cure": "重置上次睡眠时间，你不再需要睡觉了……",
+  "tooltip.biomancy.cleansing_serum": "移除所有药水效果,移除肉块中所有遗传物质。",
+  "tooltip.biomancy.decay_agent": "引发坏死，可以强烈影响僵尸。",
+  "tooltip.biomancy.mutagen_serum": "诱导肉块和其他一些生物（村民、动物、守护者）发生突变。",
+  "tooltip.biomancy.blood_sample": "含有遗传物质,把它和两剂诱变剂向肉块注射。",
+  "_tooltip.biomancy.fleshborn_door": "",
+  "tooltip.biomancy.fleshborn_trapdoor": "自动重置。\n潜行右击可以更改模式。",
+  "tooltip.biomancy.fleshborn_pressure_plate": "潜行单击以切换模式\n只有拥有者才能激活。",
+  "tooltip.biomancy.fleshborn_chest": "破坏后仍会保存内部的物品。",
+  "tooltip.biomancy.gulge": "可以储存32k相同物品，在被采集时保存其中物品。",
+  "tooltip.biomancy.chewer": "可以将把物品咀嚼成小块。\n需要提供营养来进行咀嚼。",
+  "tooltip.biomancy.digester": "消耗水将有机物变成营养物质的生物机器。\n可以从附着的容器中提取有机物。",
+  "tooltip.biomancy.solidifier": "一种可以将流体凝结成固体的分泌腺。\n可以从附着的容器或机器中提取流体。",
+  "tooltip.biomancy.decomposer": "一种可以分解物品的生物机器。\n需要提供营养来进行分解。",
+  "tooltip.biomancy.evolution_pool": "充满诱变胆汁的血肉池，用于制作有机物品。",
+  "tooltip.biomancy.boomling_hive_gun": "[WIP]与药水在工作台合成。使用营养物质重新装弹。",
+
+  "tooltip.biomancy.mining_bonus": "当挖掘 %1$s：",

```

|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
  "tooltip.biomancy.mining_bonus": "当挖掘%1$s时：",

```
如果占位符肯定是数字或英文，那么用空格，如果不是，比较建议不加空格。